deploy/eck-resources/charts/elasticsearch/values.yaml

```diff
@@ -0,0 +1,47 @@
+# TODO:
+# 1) Verify Openshift support
+# 2) Verify support of PSPs/SCCs
+
+# Whether to install an Elasticsearch cluster.
+#
+enabled: true
```

Comment:
Left over from previous version? IIUC this is only used in the parent chart in the condition and does not need to be included here?